src/components/Modal/Modal.tsx

```diff
@@ -57,8 +60,8 @@ export const ModalMeta = (props: PropsOverride) => {
       title: props.title,
       variant: props.variant,
       width: props.width,
-      withoutClose: props.withoutClose,
```

Comment:
proc se mazal `withoutClose` prop?

Reply:
Protoze tento prop z ModalMeta konfliktuje s tim co posila v props do Modal komponenty. Proto to je vzdy undefined i kdyz poslu true. Navic, tento prop se nepouizva v ModalMeta, proto byl smazany